public/components/integrations/components/added_integration_table.tsx

```diff
@@ -94,7 +94,7 @@ export function AddedIntegrationsTable(props: AddedIntegrationsTableProps) {
       truncateText: true,
       render: (value, record) => (
         <EuiText data-test-subj={`${record.templateName}IntegrationDescription`}>
-          {truncate(record.dataSourceMDSLabel || 'Local cluster', { length: 100 })}
+          {_.truncate(record.dataSourceMDSLabel || 'Local cluster', { length: 100 })}
```

Comment:
there was a PR to remove importing full lodash library, does this increase loaded javascript size for users?

Reply:
That lodash PR wasn't backported, hence there was this import issue when the MDS PR was being backported.